QUARKUS-2489.md

```diff
@@ -0,0 +1,53 @@
+# QUARKUS-2489 OIDC Back channel logout
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2489
+
+This feature is talking about logout flow on an OIDC provider as Keycloak
+
+## Scope of the testing
+
+Backchannel logout applies to `quarkus-oidc` and basically is a standard way to do logout in to a single sign-on service.
+We will use Keycloak as an OIDC provider and the flavor of the extensions (classic or reactive) is not relevant for this coverage. 
```

Comment:
you mean resteasy extensions?

Reply:
While OIDC layer that handles this before RESTEasy classic/reactive, what happens when things goes wrong is very much not. F.e. PR added code that throws `InvalidJwtException` (it's caught later, but that's implementation detail that can easily change), and the extensions handle these situations via their own mechanisms. IMHO complex test for Reactive is fine, but POC for classic (some minimal scenario) would be nice.

Reply:
thanks for input, this makes sense, there should be some coverage for resteasy classic